build.gradle

```diff
@@ -18,19 +19,15 @@ buildscript {
 }
 
 allprojects {
-	repositories {
-		google()
-		mavenCentral()
-		maven { url 'https://jitpack.io' }
-	}
```

Comment:
Why move this block in `settings.gradle`? 🤔 And have it there two times for `pluginManagement` and `dependencyResolutionManagement`?

Reply:
This is the modern way to do it in Android apps. In fact, the top-level build file should only define plugins and nothing else. Defining dependencies is the job of `settings.gradle`.

Reply:
Ah, okay, I see 👍